content/en/firewall.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Firewall
+status: Feedback Appreciated
+category: Technology
+---
+
+## What it is
+A firewall is a system that filters network traffic to protect against malware and network-based threats. It can either let traffic pass or block it based on specified rules. Firewalls can be hardware, software, or a combination of the two.
```

Comment:
Drop the "to protect against" line. It's a little misleading about the purpose of a firewall.